vk/backend/backend.go

```diff
@@ -101,11 +102,29 @@ func RunWithBackend(ctx context.Context, runner *runner.Runner, statuses *os.Fil
 	}
 
 	requests := pod.Spec.Containers[0].Resources.Requests
-	disk := requests["titus/disk"]
-	cpu := requests["cpu"]
-	memory := requests["memory"]
 
-	err = runner.StartTask(pod.GetName(), &containerInfo, memory.Value(), cpu.Value(), uint64(disk.Value()))
+	// TODO: pick one, agreed upon resource name after migration to k8s scheduler is complete.
+	disk, _ := resource.ParseQuantity("2G")
+	for _, k := range []v1.ResourceName{"titus/disk", v1.ResourceEphemeralStorage, v1.ResourceStorage} {
```

Comment:
What do you think about checking for the k8s resource (`ephemeral-storage`?) first, and fall back to `titus/disk` if it's not present?  That way, it will work with the k8s scheduler.

Reply:
Just saw your other PR - apologies for the noise in this one.

Reply:
Yup, I pushed a change that always checks k8s resource names first.